config/docker/chroot.Dockerfile

```diff
@@ -13,9 +13,10 @@
 # limitations under the License.
 FROM ubuntu:19.10
 
-RUN apt-get update && apt-get install -y debootstrap
-RUN apt-get upgrade -y
-
-RUN debootstrap --variant minbase --include python3 eoan /chroot
-RUN chroot /chroot /usr/sbin/useradd --no-create-home -u 1000 user
-RUN /usr/sbin/useradd --no-create-home -u 1000 user
+RUN apt-get update \
+    && apt-get install -yq --no-install-recommends debootstrap \
+    && rm -rf /var/lib/apt/lists/* \
+    && debootstrap --variant minbase --include python3 eoan /chroot \
```

Comment:
is this all a single command so that the apt-get remove can clean up all files from the layer?

Reply:
Yep, since we will only need the artifacts generated by the Dockerfile, this makes sure the image just contains what it needs.